assets/sass/patterns/molecules/site-header-nav-bar-secondary.scss

```diff
@@ -29,8 +29,8 @@
   @include padding(0 12 0 0);
   text-transform: uppercase;
 
-  &:last-child {
-    align-self: self-end;
+  &.nav-secondary__item--logged-in {
+    align-self: stretch;
```

Comment:
This should only apply when JavaScript is available, we still want `baseline` cross axis alignment for the text displayed without JavaScript. Suggest the selector is modified to something like:

```.js &.nav-secondary__item--logged-in```